packages/devtools/src/integrations/vscode.ts

```diff
@@ -64,21 +85,23 @@ export async function setup({ nuxt, options, openInEditorHooks, rpc }: NuxtDevto
 
     // Install VS Code Server Controller
     // https://github.com/antfu/vscode-server-controller
-    execa('code-server', [
-      'serve-local',
-      '--accept-server-license-terms',
+    execa(codeBinary, [
+      launchArg,
+      licenseTermsArg,
       '--install-extension',
       'antfu.vscode-server-controller',
+      '--host=0.0.0.0',
```

Comment:
I'd be worried that this would put the dev server at risk. The default should be safe, and we could provide an option to configure the host for containers env.

Reply:
Good point! Yes, will do!

Reply:
Updated!